docs/interactions/Receiving_and_Responding.md

```diff
@@ -241,6 +241,7 @@ Not all message fields are currently supported.
 | flags?            | integer                                                                          | [message flags](#DOCS_RESOURCES_CHANNEL/message-object-message-flags) combined as a [bitfield](https://en.wikipedia.org/wiki/Bit_field) (only `SUPPRESS_EMBEDS`, `EPHEMERAL`, and `SUPPRESS_NOTIFICATIONS` can be set) |
 | components?       | array of [components](#DOCS_INTERACTIONS_MESSAGE_COMPONENTS/)                    | message components                                                                                                                                                                                                     |
 | attachments? \*   | array of partial [attachment](#DOCS_RESOURCES_CHANNEL/attachment-object) objects | attachment objects with filename and description                                                                                                                                                                       |
+| poll?             | [poll](#DOCS_RESOURCES_POLL/poll-create-request-object) request object           | A poll!                                                                                                                                                                                                                |
```

Comment:
The description here seems to completely mismatch every other description in this table. Same with every other table modified.

Reply:
I found the description kind of redundant since the link to the poll object covers everything that is necessary, so I chose to add a little flair.

Reply:
Then it could be at least lowercase like the others